### ISSUE TITLE
Rework restart modes

### ISSUE THREAD

**Issue by maximumstock** [maximumstock/poe-stash-indexer#31]
Support both indexer restart modes:

1. wait + restart + resume
2. wait + restart + start fresh

However, both modes seem to be flawed:
1. Resuming results in inaccurate `created_at` timestamps, so its impossible to know when a new stash update was issued. As this is impossible without a whitelist record anyway, this might be the most suitable way, though.
2. There is no definitive way of getting the lastest change id. poe.ninja has an endpoint for this, but (at least for me) its currently unclear if their data is actually up-to-date or how long that datapoint lags behind the official stream.

So basically, a fresh start to collect accurate timestamps is impossible without being whitelisted by GGG, therefore mode 1 is the only viable way for the non-whitelisted rest of us. We should support both, but default to mode 1.
